cse-cc-export-runner-app/src/main/java/com/farao_community/farao/cse/export_runner/app/services/FileImporter.java

```diff
@@ -42,17 +45,29 @@ Network importNetwork(String cgmUrl) {
         return Network.read(FileUtil.getFilenameFromUrl(cgmUrl), openUrlStream(cgmUrl));
     }
 
-    CseCrac importCseCrac(String cracUrl) {
+    CRACDocumentType importCseCrac(String cracUrl) {
         InputStream cracInputStream = openUrlStream(cracUrl);
-        CseCracImporter cseCracImporter = new CseCracImporter();
-        return cseCracImporter.importNativeCrac(cracInputStream);
+        return importNativeCrac(cracInputStream);
+    }
+
+    CRACDocumentType importNativeCrac(InputStream inputStream) {
+        CRACDocumentType cracDocumentType;
+        try {
+            cracDocumentType = JAXBContext.newInstance(CRACDocumentType.class)
```

Comment:
You could have put the return directly inside of the try and remove the useless variable declaration and assignment.